Failed: opcap-weekly-audit-4.10-c9jnx Aug 12, 2022, 6:48 PM

I0812 23:48:34.212112       7 request.go:601] Waited for 1.044313543s due to client-side throttling, not priority and fairness, request: GET:https://api.audit-4-10.opdev.io:6443/apis/psmdb.percona.com/v1-1-0?timeout=32s
check called
I0812 23:48:44.261340       7 request.go:601] Waited for 4.996216696s due to client-side throttling, not priority and fairness, request: GET:https://api.audit-4-10.opdev.io:6443/apis/maps.k8s.elastic.co/v1alpha1?timeout=32s

opcap report:
-----------------------------------------
Report Date: 2022-08-12 23:49:45.819435751 +0000 UTC m=+77.723441312
OpenShift Version: 4.10.27
Package Name: smilecdr-operator
Channel: alpha
Catalog Source: certified-operators
Install Mode: SingleNamespace
Result: timeout
Message: installing: waiting for deployment smilecdr-operator-controller-manager to become ready: deployment "smilecdr-operator-controller-manager" not available: Deployment does not have minimum availability.
Reason: InstallWaiting
-----------------------------------------
exiting OperandInstall since CSV has failed

opcap report:
-----------------------------------------
Report Date: 2022-08-12 23:51:43.232273039 +0000 UTC m=+195.136278603
OpenShift Version: 4.10.27
Package Name: dynatrace-operator
Channel: alpha
Catalog Source: certified-operators
Install Mode: AllNamespaces
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand failed to create: dynakubes package: dynatrace-operator error: admission webhook "webhook.dynatrace.com" denied the request: 
1 error(s) found in the Dynakube
 1. The DynaKube's specification is missing the API URL or still has the example value set.
	Make sure you correctly specify the URL in your custom resource.
	

opcap report:
-----------------------------------------
Report Date: 2022-08-12 23:52:10.186788651 +0000 UTC m=+222.090794213
OpenShift Version: 4.10.27
Package Name: intel-device-plugins-operator
Channel: alpha
Catalog Source: certified-operators
Install Mode: AllNamespaces
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand failed to create: sgxdeviceplugins package: intel-device-plugins-operator error: the server could not find the requested resource

opcap report:
-----------------------------------------
Report Date: 2022-08-12 23:52:49.805934128 +0000 UTC m=+261.709939690
OpenShift Version: 4.10.27
Package Name: ibm-security-verify-operator
Channel: stable
Catalog Source: certified-operators
Install Mode: AllNamespaces
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand failed to create: ibmsecurityverifies package: ibm-security-verify-operator error: admission webhook "vibmsecurityverify.kb.io" denied the request: The spec.clientSecret field, --secret--, does not correspond to an available secret in the opcap-ibm-security-verify-operator namespace.

opcap report:
-----------------------------------------
Report Date: 2022-08-12 23:53:05.701243854 +0000 UTC m=+277.605249407
OpenShift Version: 4.10.27
Package Name: ibm-block-csi-operator
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: ibmblockcsis package: ibm-block-csi-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-12 23:53:45.293592175 +0000 UTC m=+317.197597737
OpenShift Version: 4.10.27
Package Name: rookout-k8s-operator
Channel: alpha
Catalog Source: certified-operators
Install Mode: AllNamespaces
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand failed to create: rookouts package: rookout-k8s-operator error: Rookout.rookout.rookout.com "rookout-operator-configuration" is invalid: spec.matchers.labels: Invalid value: "string": spec.matchers.labels in body must be of type object: "string"

opcap report:
-----------------------------------------
Report Date: 2022-08-12 23:54:10.952680043 +0000 UTC m=+342.856685611
OpenShift Version: 4.10.27
Package Name: antrea-operator-for-kubernetes
Channel: alpha
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand failed to create: antreainstalls package: antrea-operator-for-kubernetes error: AntreaInstall.operator.antrea.vmware.com "antrea-install" is invalid: spec.antreaPlatform: Required value

opcap report:
-----------------------------------------
Report Date: 2022-08-12 23:54:37.555476984 +0000 UTC m=+369.459482547
OpenShift Version: 4.10.27
Package Name: aikit-operator
Channel: alpha
Catalog Source: certified-operators
Install Mode: AllNamespaces
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: aikitcontainers package: aikit-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-12 23:55:00.286279886 +0000 UTC m=+392.190285448
OpenShift Version: 4.10.27
Package Name: crunchy-postgres-operator
Channel: v5
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand failed to create: postgresclusters package: crunchy-postgres-operator error: PostgresCluster.postgres-operator.crunchydata.com "example" is invalid: spec.backups: Required value

opcap report:
-----------------------------------------
Report Date: 2022-08-12 23:55:48.78996907 +0000 UTC m=+440.693974631
OpenShift Version: 4.10.27
Package Name: cnfv-operator
Channel: alpha
Catalog Source: certified-operators
Install Mode: AllNamespaces
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: cloudnfvs package: cnfv-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-12 23:56:13.685030358 +0000 UTC m=+465.589035921
OpenShift Version: 4.10.27
Package Name: pmem-csi-operator-os
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand failed to create: pmemcsideployments package: pmem-csi-operator-os error: the server could not find the requested resource

opcap report:
-----------------------------------------
Report Date: 2022-08-12 23:56:44.35435724 +0000 UTC m=+496.258362802
OpenShift Version: 4.10.27
Package Name: runtime-component-operator-certified
Channel: beta2
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: runtimecomponents package: runtime-component-operator-certified

opcap report:
-----------------------------------------
Report Date: 2022-08-12 23:57:46.978252989 +0000 UTC m=+558.882258550
OpenShift Version: 4.10.27
Package Name: k8s-triliovault-ibm
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: timeout
Message: installing: waiting for deployment k8s-triliovault-control-plane to become ready: deployment "k8s-triliovault-control-plane" not available: Deployment does not have minimum availability.
Reason: InstallWaiting
-----------------------------------------
exiting OperandInstall since CSV has failed

opcap report:
-----------------------------------------
Report Date: 2022-08-12 23:59:23.810656496 +0000 UTC m=+655.714662051
OpenShift Version: 4.10.27
Package Name: citrix-cpx-istio-sidecar-injector-operator
Channel: alpha
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: citrix-cpx-istio-sidecar-injectors package: citrix-cpx-istio-sidecar-injector-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:00:02.63608376 +0000 UTC m=+694.540089322
OpenShift Version: 4.10.27
Package Name: gitlab-operator-kubernetes
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: certificates package: gitlab-operator-kubernetes

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:01:06.047299305 +0000 UTC m=+757.951304854
OpenShift Version: 4.10.27
Package Name: app-director-operator
Channel: 0.7.4
Catalog Source: certified-operators
Install Mode: AllNamespaces
Result: timeout
Message: installing: waiting for deployment app-director-controller-manager to become ready: deployment "app-director-controller-manager" not available: Deployment does not have minimum availability.
Reason: InstallWaiting
-----------------------------------------
exiting OperandInstall since CSV has failed

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:02:40.038320625 +0000 UTC m=+851.942326186
OpenShift Version: 4.10.27
Package Name: nvidia-network-operator
Channel: v1.2.1
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand failed to create: hostdevicenetworks package: nvidia-network-operator error: the server could not find the requested resource

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:03:36.073340772 +0000 UTC m=+907.977346337
OpenShift Version: 4.10.27
Package Name: infoscale-operator
Channel: stable
Catalog Source: certified-operators
Install Mode: AllNamespaces
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: issuers package: infoscale-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:04:41.601274143 +0000 UTC m=+973.505279710
OpenShift Version: 4.10.27
Package Name: k8s-triliovault
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: timeout
Message: installing: waiting for deployment k8s-triliovault-control-plane to become ready: deployment "k8s-triliovault-control-plane" not available: Deployment does not have minimum availability.
Reason: InstallWaiting
-----------------------------------------
exiting OperandInstall since CSV has failed

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:06:43.665261851 +0000 UTC m=+1095.569267412
OpenShift Version: 4.10.27
Package Name: infinibox-operator-certified
Channel: stable
Catalog Source: certified-operators
Install Mode: AllNamespaces
Result: timeout
Message: installing: waiting for deployment controller-manager to become ready: deployment "controller-manager" not available: Deployment does not have minimum availability.
Reason: InstallWaiting
-----------------------------------------
operand succeeded: infiniboxcsidrivers package: infinibox-operator-certified

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:08:04.646251511 +0000 UTC m=+1176.550257068
OpenShift Version: 4.10.27
Package Name: installkit-customer
Channel: stable
Catalog Source: certified-operators
Install Mode: AllNamespaces
Result: timeout
Message: installing: waiting for deployment runai-operator to become ready: deployment "runai-operator" not available: Deployment does not have minimum availability.
Reason: InstallWaiting
-----------------------------------------
exiting OperandInstall since no ALM_Examples found in CSV

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:08:39.730255898 +0000 UTC m=+1211.634261460
OpenShift Version: 4.10.27
Package Name: cloudbees-ci
Channel: alpha
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: cloudbeescis package: cloudbees-ci

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:09:05.776274685 +0000 UTC m=+1237.680280246
OpenShift Version: 4.10.27
Package Name: hpe-csi-operator
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: hpecsidrivers package: hpe-csi-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:10:08.664505277 +0000 UTC m=+1300.568510837
OpenShift Version: 4.10.27
Package Name: federatorai-certified
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: timeout
Message: installing: waiting for deployment federatorai-operator to become ready: deployment "federatorai-operator" not available: Deployment does not have minimum availability.
Reason: InstallWaiting
-----------------------------------------
exiting OperandInstall since CSV has failed

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:11:33.475943785 +0000 UTC m=+1385.379949347
OpenShift Version: 4.10.27
Package Name: anzounstructured-operator
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: anzounstructureds package: anzounstructured-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:12:36.399247589 +0000 UTC m=+1448.303253150
OpenShift Version: 4.10.27
Package Name: redhat-marketplace-operator
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: timeout
Message: installing: waiting for deployment redhat-marketplace-controller-manager to become ready: deployment "redhat-marketplace-controller-manager" waiting for 1 outdated replica(s) to be terminated
Reason: InstallWaiting
-----------------------------------------
operand succeeded: marketplaceconfigs package: redhat-marketplace-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:13:19.8469319 +0000 UTC m=+1491.750937463
OpenShift Version: 4.10.27
Package Name: citrix-ingress-controller-operator
Channel: alpha
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: citrixingresscontrollers package: citrix-ingress-controller-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:13:59.036318436 +0000 UTC m=+1530.940323998
OpenShift Version: 4.10.27
Package Name: mongodb-atlas-kubernetes
Channel: beta
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand failed to create: atlasbackuppolicies package: mongodb-atlas-kubernetes error: AtlasBackupPolicy.atlas.mongodb.com "atlasbackuppolicy-sample" is invalid: [spec.items.frequencyType: Unsupported value: "WEEKLY": supported values: "hourly", "daily", "weekly", "monthly", spec.items.retentionUnit: Unsupported value: "DAYS": supported values: "days", "weeks", "months"]

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:14:25.698258083 +0000 UTC m=+1557.602263645
OpenShift Version: 4.10.27
Package Name: open-liberty-certified
Channel: beta2
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: openlibertyapplications package: open-liberty-certified

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:15:27.24063543 +0000 UTC m=+1619.144640992
OpenShift Version: 4.10.27
Package Name: tenant-operator
Channel: alpha
Catalog Source: certified-operators
Install Mode: AllNamespaces
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand failed to create: tenants package: tenant-operator error: the server could not find the requested resource

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:16:26.289189036 +0000 UTC m=+1678.193194605
OpenShift Version: 4.10.27
Package Name: infrastructure-asset-orchestrator-certified
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: generatekeys package: infrastructure-asset-orchestrator-certified

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:17:02.433061006 +0000 UTC m=+1714.337066567
OpenShift Version: 4.10.27
Package Name: sysdig-certified
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: sysdigagents package: sysdig-certified

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:17:29.7802918 +0000 UTC m=+1741.684297359
OpenShift Version: 4.10.27
Package Name: openshiftxray-operator
Channel: alpha
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: openshiftxrays package: openshiftxray-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:18:05.030166714 +0000 UTC m=+1776.934172265
OpenShift Version: 4.10.27
Package Name: operator-certification-operator
Channel: alpha
Catalog Source: certified-operators
Install Mode: AllNamespaces
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: operatorpipelines package: operator-certification-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:18:45.160851804 +0000 UTC m=+1817.064857357
OpenShift Version: 4.10.27
Package Name: anchore-engine
Channel: alpha
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: anchoreengines package: anchore-engine

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:19:08.915280766 +0000 UTC m=+1840.819286327
OpenShift Version: 4.10.27
Package Name: kong-offline-operator
Channel: alpha
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: kongs package: kong-offline-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:19:32.419374296 +0000 UTC m=+1864.323379858
OpenShift Version: 4.10.27
Package Name: bookkeeper-operator
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: bookkeeperclusters package: bookkeeper-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:20:00.647529963 +0000 UTC m=+1892.551535525
OpenShift Version: 4.10.27
Package Name: seldon-operator-certified
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand failed to create: seldondeployments package: seldon-operator-certified error: Internal error occurred: failed calling webhook "v1.vseldondeployment.kb.io": failed to call webhook: Post "https://seldon-webhook-service.opcap-seldon-operator-certified.svc:443/validate-machinelearning-seldon-io-v1-seldondeployment?timeout=10s": dial tcp 10.128.1.82:8443: connect: connection refused

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:20:26.36986353 +0000 UTC m=+1918.273869092
OpenShift Version: 4.10.27
Package Name: t8c-certified
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: xls package: t8c-certified

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:20:51.152777359 +0000 UTC m=+1943.056782921
OpenShift Version: 4.10.27
Package Name: xcrypt-operator
Channel: alpha
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: xcrypts package: xcrypt-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:21:27.623276855 +0000 UTC m=+1979.527282416
OpenShift Version: 4.10.27
Package Name: nginx-ingress-operator
Channel: alpha
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: nginxingresses package: nginx-ingress-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:22:04.598922644 +0000 UTC m=+2016.502928206
OpenShift Version: 4.10.27
Package Name: kubeturbo-certified
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: kubeturbos package: kubeturbo-certified

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:22:28.416305891 +0000 UTC m=+2040.320311443
OpenShift Version: 4.10.27
Package Name: hpe-ezmeral-csi-operator
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: hpeezmeralcsidrivers package: hpe-ezmeral-csi-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:23:06.758204003 +0000 UTC m=+2078.662209566
OpenShift Version: 4.10.27
Package Name: apimatic-codegen-operator
Channel: beta
Catalog Source: certified-operators
Install Mode: AllNamespaces
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: codegens package: apimatic-codegen-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:23:37.691931241 +0000 UTC m=+2109.595936802
OpenShift Version: 4.10.27
Package Name: pulsar-operator
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand failed to create: pulsarbrokers package: pulsar-operator error: PulsarBroker.pulsar.streamnative.io "pulsarbroker-sample" is invalid: spec.zkServers: Required value

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:24:04.352306656 +0000 UTC m=+2136.256312240
OpenShift Version: 4.10.27
Package Name: neuvector-certified-operator
Channel: beta
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: neuvectors package: neuvector-certified-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:24:36.491844497 +0000 UTC m=+2168.395850060
OpenShift Version: 4.10.27
Package Name: sanstoragecsi-operator-bundle
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: sanstoragecsis package: sanstoragecsi-operator-bundle

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:25:09.378468944 +0000 UTC m=+2201.282474507
OpenShift Version: 4.10.27
Package Name: ako-operator
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand failed to create: akoconfigs package: ako-operator error: AKOConfig.ako.vmware.com "ako-sample" is invalid: [spec.controllerSettings.tenantsPerCluster: Invalid value: "string": spec.controllerSettings.tenantsPerCluster in body must be of type boolean: "string", spec.akoSettings.enableEvents: Invalid value: "string": spec.akoSettings.enableEvents in body must be of type boolean: "string", spec.akoSettings.vipPerNamespace: Invalid value: "string": spec.akoSettings.vipPerNamespace in body must be of type boolean: "string"]

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:25:48.062312846 +0000 UTC m=+2239.966318424
OpenShift Version: 4.10.27
Package Name: entando-k8s-operator
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: entandodebundles package: entando-k8s-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:26:27.437328633 +0000 UTC m=+2279.341334195
OpenShift Version: 4.10.27
Package Name: cass-operator
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: cassandradatacenters package: cass-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:26:50.174185962 +0000 UTC m=+2302.078191534
OpenShift Version: 4.10.27
Package Name: mongodb-enterprise
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: mongodb package: mongodb-enterprise

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:27:17.934040316 +0000 UTC m=+2329.838045877
OpenShift Version: 4.10.27
Package Name: anzograph-operator
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: anzographs package: anzograph-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:28:22.503627595 +0000 UTC m=+2394.407633147
OpenShift Version: 4.10.27
Package Name: redis-enterprise-operator-cert
Channel: production
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: timeout
Message: installing: waiting for deployment redis-enterprise-operator to become ready: deployment "redis-enterprise-operator" not available: Deployment does not have minimum availability.
Reason: InstallWaiting
-----------------------------------------
exiting OperandInstall since CSV has failed

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:30:09.161195377 +0000 UTC m=+2501.065200938
OpenShift Version: 4.10.27
Package Name: silicom-sts-operator
Channel: alpha
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand failed to create: stsconfigs package: silicom-sts-operator error: StsConfig.sts.silicom.com "gm-1" is invalid: spec.interfaces.ethPort: Invalid value: 0: spec.interfaces.ethPort in body should be greater than or equal to 1

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:30:39.506308885 +0000 UTC m=+2531.410314447
OpenShift Version: 4.10.27
Package Name: kubemq-operator-marketplace
Channel: alpha
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: kubemqclusters package: kubemq-operator-marketplace

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:31:06.541274695 +0000 UTC m=+2558.445280256
OpenShift Version: 4.10.27
Package Name: gitlab-runner-operator
Channel: stable
Catalog Source: certified-operators
Install Mode: AllNamespaces
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand failed to create: runners package: gitlab-runner-operator error: Internal error occurred: error resolving resource

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:32:11.788274001 +0000 UTC m=+2623.692279563
OpenShift Version: 4.10.27
Package Name: minio-operator
Channel: stable
Catalog Source: certified-operators
Install Mode: AllNamespaces
Result: timeout
Message: installing: waiting for deployment minio-operator to become ready: deployment "minio-operator" not available: Deployment does not have minimum availability.
Reason: InstallWaiting
-----------------------------------------
exiting OperandInstall since CSV has failed

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:33:43.546869942 +0000 UTC m=+2715.450875503
OpenShift Version: 4.10.27
Package Name: couchbase-enterprise-certified
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: couchbaseclusters package: couchbase-enterprise-certified

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:34:19.836748201 +0000 UTC m=+2751.740753765
OpenShift Version: 4.10.27
Package Name: infoscale-sds-operator
Channel: stable
Catalog Source: certified-operators
Install Mode: AllNamespaces
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand failed to create: licenses package: infoscale-sds-operator error: the server could not find the requested resource

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:34:56.393268926 +0000 UTC m=+2788.297274488
OpenShift Version: 4.10.27
Package Name: nxrm-operator-certified
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: nexusrepos package: nxrm-operator-certified

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:35:44.869000277 +0000 UTC m=+2836.773005839
OpenShift Version: 4.10.27
Package Name: pcc-operator
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: consoles package: pcc-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:36:12.967233984 +0000 UTC m=+2864.871239546
OpenShift Version: 4.10.27
Package Name: aqua-operator-certified
Channel: 2022.4
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: aquacsps package: aqua-operator-certified

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:37:18.06298872 +0000 UTC m=+2929.966994292
OpenShift Version: 4.10.27
Package Name: memcached-operator-ogaye
Channel: alpha
Catalog Source: certified-operators
Install Mode: AllNamespaces
Result: timeout
Message: installing: waiting for deployment memcached-operator-controller-manager to become ready: deployment "memcached-operator-controller-manager" not available: Deployment does not have minimum availability.
Reason: InstallWaiting
-----------------------------------------
exiting OperandInstall since CSV has failed

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:38:58.001100943 +0000 UTC m=+3029.905106542
OpenShift Version: 4.10.27
Package Name: hazelcast-platform-operator
Channel: alpha
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: hazelcasts package: hazelcast-platform-operator

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:39:47.910077492 +0000 UTC m=+3079.814083060
OpenShift Version: 4.10.27
Package Name: gpu-operator-certified
Channel: v1.11
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand failed to create: clusterpolicies package: gpu-operator-certified error: the server could not find the requested resource

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:40:27.735339156 +0000 UTC m=+3119.639344718
OpenShift Version: 4.10.27
Package Name: prometurbo-certified
Channel: stable
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: prometurbos package: prometurbo-certified

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:40:58.787068737 +0000 UTC m=+3150.691074300
OpenShift Version: 4.10.27
Package Name: datadog-operator-certified
Channel: alpha
Catalog Source: certified-operators
Install Mode: OwnNamespace
Result: Succeeded
Message: install strategy completed with no errors
Reason: InstallSucceeded
-----------------------------------------
operand succeeded: datadogagents package: datadog-operator-certified

opcap report:
-----------------------------------------
Report Date: 2022-08-13 00:42:03.478256665 +0000 UTC m=+3215.382262226
OpenShift Version: 4.10.27
Package Name: rtf-agent-operator
Channel: alpha
Catalog Source: certified-operators
Install Mode: AllNamespaces
Result: timeout
Message: installing: waiting for deployment rtf-agent-operator-controller-manager to become ready: deployment "rtf-agent-operator-controller-manager" not available: Deployment does not have minimum availability.
Reason: InstallWaiting
-----------------------------------------
exiting OperandInstall since CSV has failed
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/opdev/opcap/internal/capability.(*CapAudit).getAlmExamples(0xc0003ef080)
	/home/mrhillsman/Development/goland/opcap/internal/capability/operand_install.go:36 +0x20b
github.com/opdev/opcap/internal/capability.(*CapAudit).OperandInstall(0xc0003ef080)
	/home/mrhillsman/Development/goland/opcap/internal/capability/operand_install.go:54 +0x1b7
reflect.Value.call({0x1804be0, 0xc0003ef080, 0x1}, {0x197fbb0, 0x4}, {0x0, 0x0, 0x1c57ef0})
	/usr/local/go/src/reflect/value.go:556 +0x845
reflect.Value.Call({0x1804be0, 0xc0003ef080, 0x4a}, {0x0, 0x0, 0x0})
	/usr/local/go/src/reflect/value.go:339 +0xc5
github.com/opdev/opcap/internal/capability.(*capAuditor).RunAudits(0xc00012bd68)
	/home/mrhillsman/Development/goland/opcap/internal/capability/auditor.go:84 +0x1bd
github.com/opdev/opcap/cmd.glob..func2(0x2997f80, {0x19800b8, 0x2, 0x2})
	/home/mrhillsman/Development/goland/opcap/cmd/check.go:79 +0x112
github.com/spf13/cobra.(*Command).execute(0x2997f80, {0xc0004681c0, 0x2, 0x2})
	/home/mrhillsman/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:860 +0x5f8
github.com/spf13/cobra.(*Command).ExecuteC(0x2998200)
	/home/mrhillsman/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:974 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
	/home/mrhillsman/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:902
github.com/opdev/opcap/cmd.Execute()
	/home/mrhillsman/Development/goland/opcap/cmd/root.go:32 +0x25
main.main()
	/home/mrhillsman/Development/goland/opcap/main.go:10 +0x17

--- a/internal/capability/operand_install.go
+++ b/internal/capability/operand_install.go
@@ -32,9 +32,11 @@ func (ca *CapAudit) getAlmExamples() error {
 		return err
 	}
 
+	almExamples := ""
 	// map of string interface which consist of ALM examples from the CSVList
-	almExamples := CSVList.Items[0].ObjectMeta.Annotations["alm-examples"]
-
+	if len(CSVList.Items) > 0 {
+		almExamples = CSVList.Items[0].ObjectMeta.Annotations["alm-examples"]
+	}
 	var almList []map[string]interface{}
 
 	err = yaml.Unmarshal([]byte(almExamples), &almList)

